solutions/swb-app/src/dataSetRoutes.ts

```diff
@@ -181,6 +185,21 @@ export function setUpDSRoutes(router: Router, dataSetService: DataSetPlugin): vo
     })
   );
 
+  router.get(
+    '/datasets/:datasetId/permissions',
+    wrapAsync(async (req: Request, res: Response) => {
+      const validatedRequest = validateAndParse<ListDataSetAccessPermissionsRequest>(
+        ListDataSetAccessPermissionsRequestParser,
+        {
+          ...req.query,
```

Comment:
is it safe just to spread the query like this? Looking at the parser, you want pagination token from the query. Why not just specify that clearly?

Reply:
The parser will only take the data that it has been instructed to parse. Right now, workbench core's implementation doesn't take a page size, but I think its going to be refactored to do that. Doing it this way means I only have to make a change in the parser if we decide to accept page size in the future. I can change it though if you think its worth doing.

Reply:
Yup, that makes sense! Good with this!